### ISSUE TITLE
Elastic index configuration using type static with fluent-plugin-viaq plugin failing with ERROR

### ISSUE THREAD

**Issue by abu-2020-ghub** [openshift/origin-aggregated-logging#1957]
Trying the below configuration with fluent-plugin-viaq plugin in openshift-logging operator, but it fails with ERROR
```
<elasticsearch_index_name>
     enabled "true"
     tag "myapp*"
     name_type static
     static_index_name myapp
</elasticsearch_index_name>
```

```
[]# oc logs -c fluentd fluentd-ccb4l
2020-08-03 03:37:22 +0000 [error]: config error file="/etc/fluent/fluent.conf" error_class=Fluent::ConfigError error="valid options are operations_full,project_full,operations_prefix,project_prefix,audit_full,audit_prefix but got static"
```

Any support would help !!

**Comment by jcantrill** [comment]
It looks like you are trying to use a version of logging that does not support this setting.  This is only available as of release 4.5

**Comment by jcantrill** [comment]
https://github.com/openshift/origin-aggregated-logging/blame/master/fluentd/lib/fluent-plugin-viaq_data_model/lib/fluent/plugin/filter_viaq_data_model.rb#L144
